addons/pos_self_order/static/src/Components/Order/Order.js

```diff
@@ -0,0 +1,15 @@
+/** @odoo-module */
+
+import { useSelfOrder } from "@pos_self_order/SelfOrderService";
+import { Component } from "@odoo/owl";
+import { ProductCard } from "@pos_self_order/Components/ProductCard/ProductCard";
+import { OrderLines } from "@pos_self_order/Components/OrderLines/OrderLines";
+import { PriceDetails } from "@pos_self_order/Components/PriceDetails/PriceDetails";
+export class Order extends Component {
```

Comment:
the class name `Order` is used everywhere. Can make this distinguishable from other `Order` classes? Better for grepping the code.

Reply:
i agree. What do you think a better name would be? Now i have the `OrdersView` comp, which displays multiple `Order` components, each having  multiple `OrderLine` components.